Sources/Root/Cells/FilterCell.swift

```diff
@@ -57,13 +80,16 @@ class FilterCell: UITableViewCell {
         super.prepareForReuse()
         selectedValues = nil
         filterName = nil
+        dotView.isHidden = true
```

Comment:
You can also do `isContextFilter = false` instead.

Reply:
Aha, that is true! 😄